### ISSUE TITLE
OCIO accessibility report - meeting with Sid and follow up actions

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/onrr.gov-site#2513]
@eelzi-ONRR summary of our questions for Sid and notes
Questions for Sid
 
What is a good score?
What is average score for DOI?
Are there penalties or awards for our accessibility?
 
What about errors that aren't ours?
Are there known areas that it doesn’t pick upon?
Are there differences between alerts and violations?
 
Notes for us:
Categorize accessibilities by priority
Timeline for fixing
What do we need developer help on?
 
Interesting error
Progress bar - circular spinning thing in the logo in the browser
https://vuetifyjs.com/en/components/progress-circular/#usage

[violation spreadsheet](https://doimspp.sharepoint.com/:x:/r/sites/NRRD/_layouts/15/Doc.aspx?sourcedoc=%7B3E1F0B4A-7452-44DC-8D2A-A85991BB45A3%7D&file=OCIO%20report%20violations%20-%20Feb%20%2723.xlsx&_DSL=1&action=default&mobileredirect=true)

**Comment by eelzi-ONRR** [comment]
Sid sent me the Level Access training materials and we have issues for that and trusted tester training. So I'm closing this issue
